src/content/guides/development.md

```diff
@@ -274,7 +274,7 @@ __package.json__
     "scripts": {
       "test": "echo \"Error: no test specified\" && exit 1",
       "watch": "webpack --watch",
-+     "start": "webpack serve --open",
++     "start": "webpack serve",
```

Comment:
cc @evilebottnawi What do you think here? Shall we remove`--open` temporarily? It won't work until webpack-dev-server v4 is out according to https://github.com/webpack/webpack-cli/issues/2001#issuecomment-720236731

Reply:
Keep `--open`, I will fix it in near future (2-3 days)